prac_06/guitar.py

```diff
@@ -0,0 +1,22 @@
+"""Guitar Class"""
+
+
+class Guitar:
+    def __init__(self, name, year=0, cost=0):
+        self.name = name
+        self.year = year
+        self.cost = cost
+
+    def __str__(self):
+        return "{} ({}) : ${}".format(self.name, self.year, self.cost)
+
+    def get_age(self):
+        current_year = 2021
+        return current_year - self.year
+
+    def is_vintage(self):
+        age = self.get_age()
```

Comment:
Could reduce this to if self.get_age() >= 50: if you do not re-use the age variable.